src/shared/components/banner/index.js

```diff
@@ -24,8 +26,8 @@ class Banner extends Component {
         </div>
         <div className={ styles.wrapContainer }>
           <div className={ styles.textContainer } >
-            <FormattedMessage tagName="strong" id="bannerHighlightMessage" />
-            <FormattedMessage id="bannerMessage" />
+            <strong>{ banner.highlightMessage }</strong>
```

Comment:
Here you should do:
```js
<strong>{ messages.banner.highlightMessage }</strong>
```

This way is clear that this is a translation, as opposed to the way it is right now. Moreover,  it will be standardised across every component.